examples/bri-3/prisma/schema.prisma

```diff
@@ -23,12 +23,14 @@ model BpiSubject {
 model BpiAccount {
   id    String @id @default(uuid())
   nonce Int
+  ownerBpiSubjectAccounts BpiSubjectAccount[]
 }
 
 model BpiSubjectAccount {
   id                  String @id @default(uuid())
   creatorBpiSubjectId String
   ownerBpiSubjectId   String
+  bpiAccounts         BpiAccount[]
   CreatorBpiSubject   BpiSubject  @relation(fields: [creatorBpiSubjectId], references: [id], name: "creatorBpiSubject_fk")
```

Comment:
**c**reator

Reply:
dont understand this and comment below? if you are refering to small first letter that is ok, this is how is supposed to work

Reply:
ok, was unaware this is by convention